sample-app/src/main/java/com/sinch/sample/verification/verifications/GetReportById.java

```diff
@@ -0,0 +1,51 @@
+package com.sinch.sample.verification.verifications;
+
+import com.sinch.sample.BaseApplication;
+import com.sinch.sdk.domains.verification.models.VerificationMethodType;
+import com.sinch.sdk.domains.verification.models.VerificationReport;
+import com.sinch.sdk.domains.verification.models.requests.VerificationReportCalloutRequestParameters;
+import com.sinch.sdk.domains.verification.models.requests.VerificationReportFlashCallRequestParameters;
+import com.sinch.sdk.domains.verification.models.requests.VerificationReportRequestParameters;
+import com.sinch.sdk.domains.verification.models.requests.VerificationReportSMSRequestParameters;
+import java.io.IOException;
+import java.util.logging.Logger;
+
+public class GetReportById extends BaseApplication {
```

Comment:
I'm not sure using `Get` in the class' name is reflecting the purpose of the API as it is a 'PUT'

Reply:
Well, the action here is to "report" a verification (by submitting a `code` or a `cli`) and eventually, you get a response for this action which is indeed of the same kind of response as when you send a request to fetch a status. This is why I would rather think about `ReportWithId` or `SubmitReportWithId` as a name. WDYT?

Reply:
Yes, in this point of view, this is having sense too.
I'll change it (my preference is for `ReportWithId`)